package.json

```diff
@@ -164,6 +166,7 @@
     "build-chromeapp": "webpack --config ./scripts/chromeapp.webpack.js",
     "build-js": "webpack --config ./scripts/browser.webpack.js",
     "build-js-worker": "webpack --config ./scripts/worker.webpack.js",
+    "build-d-ts": "rm -rf dist/**/*.d.ts  &&  tsc --emitDeclarationOnly",
```

Comment:
This should likely also be added to the script `build` on line 165, e.g.
```
"build": "npm run build-js && npm run build-js-worker && npm run build-chromeapp && npm run build-d-ts",
```
To ensure it's built during a release, we'll also need to create a PR [here](https://github.com/webtorrent/semantic-release-config/blame/411b9e580dcd805eee651b91f9483d838c2a7fbd/.releaserc.js#L37) to add `dist/**/*.d.ts` to `assets` to ensure the built files are pushed back to GitHub.

Reply:
> To ensure it's built during a release, we'll also need to create a PR [here](https://github.com/webtorrent/semantic-release-config/blame/411b9e580dcd805eee651b91f9483d838c2a7fbd/.releaserc.js#L37) to add `dist/**/*.d.ts` to `assets` to ensure the built files are pushed back to GitHub.

Does it support Glob Pattern? We have lots of `.d.ts`.